diff --git a/lib/compiler/typescript-loader.js b/lib/compiler/typescript-loader.js
index 7c840defab0e01fdc2be314e3b86e49243f32e87..513e4d3c305d14afe8ebe5c57337a3061824969f 100644
--- a/lib/compiler/typescript-loader.js
+++ b/lib/compiler/typescript-loader.js
@@ -8,7 +8,7 @@ class TypeScriptBinaryLoader {
         }
         try {
             const tsBinaryPath = require.resolve('typescript', {
-                paths: [process.cwd(), ...this.getModulePaths()],
+                paths: [__dirname, ...this.getModulePaths(), process.cwd()],
             });
             // eslint-disable-next-line @typescript-eslint/no-var-requires
             const tsBinary = require(tsBinaryPath);
